package.json

```diff
@@ -372,13 +372,13 @@
     "@types/mocha": "^2.2.32",
     "@types/node": "^6.0.40",
     "cross-env": "^5.2.0",
-    "gulp": "^3.9.1",
+    "gulp": "^4.0.0",
```

Comment:
Be careful with gulp. 4.0.0 and v3.9.1 are totally different.

Reply:
yes, high risks along with the major version change.